examples/tutorials/plot_tuto_hole_generator.py

```diff
@@ -206,7 +206,7 @@ def plot_cdf(
 )
 uniform_mask = uniform_generator.split(df)[0]
 
-print("Pourcentage of additional missing values:")
+print("Percentage of additional missing values:")
```

Comment:
It's possible this is some intentional thing, but I'm hoping it isn't.

Note that there was one file that appeared to have some French -- I skipped some content there.